docs/papers/squad-paper.md

```diff
@@ -110,26 +168,45 @@ costly.
 Funding reciepts can only be transfered to addresses without any
 existing funding reciepts, and cannot be split sold or refunded.
 
+Let's consider framing this section in terms of these principles of DAO design:
+1. How and why would one become a voter?
+* Incedentally, as a result of people using (buying tokens from) your contributions
+2. How and why would one vote?
+* To fix some problem with the terms of service, which keeps the economic system you depend on safe
+* To censor some problematic content, for the same reason
+3. How and why would one fund the DAO?
+* In order to make more money off your game design contributions
+4. How and why would one get paid by the DAO?
+* Generally: for contributing to the public goods of the Squad ecosystem
+* Examples: lawyer editing the ToS or enforcing licenses, moderators identifying content that goes against ToS
+
 #### Terms of service
 
 A legally binding agreement between participants
 
-#### System Parameters
+#### System Parameters & Requirements
 
+Params:
 * Definition bond price curves
-* Reputation award amount and frequency
-* Reputation award distribution equation
+* Contributor Reputation award amount and frequency
+* Contributor Reputation award distribution equation
   * Probably logarithmic based on tokens sold
 * Gov DAO investor reputation equation and percentage
   * Probably simple token weight
+  (it can't be worse than what you'd get from buying a ton of your own contribution, or else investors might do that,
+  which is bad for the curation markets)
```

Comment:
I don't think this is a problem, as buying a ton of your own contribution is a one time event, but owning the DAO token entitles you to a fixed amount of all existing reputation. (this could be a problem if the "active voting" reputation ends up concentrated in just a few investors)

buying your own contribution will also get more expensive as more legitimate business is done

Reply:
Ah, I think I see what you're going for now... Rep calculation would be something like this?
`John's Rep = 0.2*(John's investor shares / all investor shares) + 0.8*(John's contributor shares / all contributor shares)`
That should work, though I'm not aware of any currently built DAO systems for that.